### ISSUE TITLE
Draw shaded polygons, lines, and arcs 

### ISSUE THREAD

**Issue by Ryan2516** [Stellarium/stellarium#3144]
Similar to the custom markers I would like to be able to draw shaded but transparent polygons over stars and areas in the sky. Maybe you could also be able to change the color so they could be red or purple.

**Comment by gzotti** [comment]
Yes, metoo. Semitransparent polygons show a technical problem of overlapping triangles caused by libtess.
